src/Microsoft.AspNetCore.Server.IISIntegration/Server/IISHttpServer.cs

```diff
@@ -101,8 +101,7 @@ private static NativeMethods.REQUEST_NOTIFICATION_STATUS OnAsyncCompletion(IntPt
         private static void CompleteRequest(IISHttpContext context)
         {
             // Post completion after completing the request to resume the state machine
-            //context.PostCompletion();
-            context.IndicateCompletion(NativeMethods.REQUEST_NOTIFICATION_STATUS.RQ_NOTIFICATION_CONTINUE);
+            context.PostCompletion(NativeMethods.REQUEST_NOTIFICATION_STATUS.RQ_NOTIFICATION_CONTINUE);
```

Comment:
I'm assuming this needs to change?

Reply:
What do you mean by that? I have another PR that will change the value of request notification status based on success or failure.

Reply:
NM I read the rest of the codez